pandas/core/frame.py

```diff
@@ -1598,6 +1598,29 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
+    def to_parquet(self, fname, engine=None, compression='snappy',
+                   **kwargs):
+        """
+        write out the binary parquet for DataFrames
```

Comment:
The `io.parquet.to_parquet` had a nicer docstring intro IMO, I would use that one here as well: "Write a DataFrame to the binary parquet format."